migration/src/m0000022_create_organization.rs

```diff
@@ -0,0 +1,51 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        // Replace the sample below with your own migration scripts
+        manager
+            .create_table(
+                Table::create()
+                    .table(Organization::Table)
+                    .if_not_exists()
```

Comment:
In light of #127, I'm inclined to make it a practice to avoid these guards, because it allows you to omit the `down` fn. It's a minor nit, but I like simple, correct migrations, with as little code as possible. CI will whinge if you forget the `down` now.